Sources/WeaverDI/ThreadSafeBuilderStoringDecorator.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ThreadSafeBuilderStoringDecorator.swift
+//  WeaverDIPackageDescription
+//
+//  Created by Bruno Mazzo on 5/31/18.
+//
+
+import Foundation
+
+class ThreadSafeBuilderStoringDecorator: BuilderStoring {
+    
+    let builderStoring: BuilderStoring
```

Comment:
Let's make that `private`

Reply:
Also, let's call this property `builders`, to keep it consistent with the rest of the codebase.

Reply:
Done